docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_run_command/_index.md

```diff
@@ -0,0 +1,161 @@
+---
+type: docs
+title: "Azure Arc RUN command"
+linkTitle: "Azure Arc RUN command"
+weight: 20
+description: >
+---
+
+## Run PowerShell and Shell scripts on Azure Arc-enabled servers using the Run command
+
+The Run command feature uses the Connected Machine agent to remotely run PowerShell scripts within an Azure Arc-connected Windows machine and Shell scripts within an Azure Arc-connected Linux machine. 
+
+This capability is useful in all scenarios where you want to run a script within an Arc-connected machine. It allows you to troubleshoot and remediate a machine that doesn't have the RDP or SSH port open because of improper network or administrative user configuration.
+
+The following Jumpstart scenario will guide you on how to use the Run command on your Arc-connected machine. Use the **Azure Cloud Shell** or **Visual Studio Code** command terminal to follow the instructions in this scenario.
```

Comment:
```suggestion
The Run command feature uses the Connected Machine agent to remotely run PowerShell scripts within an Azure Arc-connected Windows machine and Shell scripts within an Azure Arc-connected Linux machine. This capability is useful in all scenarios where you want to run a script within an Arc-connected machine. It allows you to troubleshoot and remediate a machine that doesn't have the RDP or SSH port open because of improper network or administrative user configuration.

The following Jumpstart scenario will guide you on how to use the Run command on your Arc-connected machine. Use the **Azure Cloud Shell** or **Visual Studio Code** command terminal to follow the instructions in this scenario.
```

Reply:
done